NDB.Covid19/NDB.Covid19.Droid/Resources/layout/activity_daily_numbers.xml

```diff
@@ -294,147 +289,313 @@
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintBottom_toBottomOf="parent">
+                    android:id="@+id/linearLayout5"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    app:layout_constraintTop_toBottomOf="@id/table4"
+                    android:layout_marginTop="@dimen/defaultVerticalMargin"
+                    android:layout_marginBottom="@dimen/defaultVerticalMargin">
 
                 <TextView
-                        android:id="@+id/daily_numbers_sub_text"
-                        style="@style/DailyNumbersSubText"
+                        android:id="@+id/daily_numbers_smittestopp_header_textView"
+                        style="@style/PrimaryTextBold"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_below="@+id/table4"
+                        android:text="Sub header"/>
+
+                <TextView
+                        android:id="@+id/daily_numbers_smittestopp_text_textview"
+                        style="@style/PrimaryTextRegular"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:layout_margin="0dp"
-                        android:layout_marginEnd="18dp"
-                        android:text="*Sub text" />
+                        android:layout_margin="0dp"        
+                        android:text="Smttestopp text" />
             </LinearLayout>
+            <TableLayout
+                    android:id="@+id/table5"
+                    android:stretchColumns="*"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:paddingVertical="@dimen/verticalPaddingDailyNumbersTable"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/linearLayout5">
+
+                <TableRow
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content">
+
+                    <TextView
+                            android:id="@+id/daily_numbers_smittestopp_downloads_header_text"
+                            style="@style/DailyNumbersItemHeaderText"
+                            android:layout_weight="0.5"
+                            android:layout_width="0dp"
+                            android:layout_height="match_parent"
+                            android:layout_column="0"
+                            android:layout_marginEnd="16dp"
```

Comment:
I see that you are using this "...marginEND = 16 dp" a couple of places. Please create a defualt dimen for this